docs/core/linux-prerequisites.md

```diff
@@ -20,9 +20,13 @@ This article shows the dependencies needed to develop .NET Core applications on
 
 ## Supported Linux versions
 
+# [.NET Core 2.x](#tab/netcore2x)
+
 .NET Core 2.x is supported on the following Linux distributions/versions:
 
- * Red Hat Enterprise Linux 7 x64
+> [!NOTE]
+> Red Hat will be providing a distribution of .NET Core 2.0 very soon. For more information, see [the Microsoft .NET download center](https://www.microsoft.com/net/core#linuxredhat).
+
  * CentOS 7 x64
  * Oracle Linux 7 x64
  * Fedora 25, 26 x64
```

Comment:
If this was Windows, we'd call out the full name Windows 7, Windows 8, etc. I think it'd be better to call out each one: Fedora 25, Fedora 26